videoclip.lua

```diff
@@ -165,6 +170,37 @@ local function upload_to_catbox(outfile)
     p.copy_or_open_url(r.stdout)
 end
 
+local function upload_to_custom(outfile)
+    h.notify("Upload to custom destination", "info", 9999)
+
+    -- Replace %f with the file path
+    local command = config.custom_upload_command:gsub('%%f', outfile)
+
+    -- Execute the command through the shell
+    local shell = h.is_win() and 'cmd.exe' or 'sh'
+    local shell_arg = h.is_win() and '/C' or '-c'
+    local r = h.subprocess({ shell, shell_arg, command })
```

Comment:
You can pass arguments directly to subprocess instead of using a shell. It can be done the same way `autoclip_custom_args` is handled in mpvacious. https://github.com/Ajatt-Tools/mpvacious/blob/eb639e0115397412d7c917a237dd8a0b788ea4c0/subtitles/observer.lua#L90

Reply:
Sure, but this still leaves out most features of a shell, like for example command substitution which i also personally take advantage of in my custom command. I think its unreasonable to reimplement all features of a shell so i will just leave it as it is right now (with arg parsing)

Reply:
I was gonna push a commit with updated code but theres currently a github outage :smile: 
<img width="596" height="153" alt="image" src="https://github.com/user-attachments/assets/190c8b78-b196-4afc-ac9c-999132daa4b6" />